curtsies/escseqparse.py

```diff
@@ -66,7 +66,7 @@ def peel_off_esc_code(s):
         del d['front']
         del d['rest']
```

Comment:
Looks like it's so all the interesting bits are in the dictionary d refers to (which is distinct from the m.groupdict() one, `m.groupdict()` makes a new one each time). I guess the (undocumented) API is for the second argument of the returned tuple not to have those in there.